catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -178,10 +189,29 @@ public Object visit(PropertyIsEqualTo filter, Object extraData) {
             sourceIds.add((String) ((Literal) filter.getExpression2()).getValue());
             return null;
         }
+        AttributeType type =
```

Comment:
You need to do the same logic for other operations like: `between, `lessThan`, `greaterThan`, etc.